src/code_80057C60.c

```diff
@@ -5,9 +5,15 @@
 #include <variables.h>
 #include <config.h>
 #include <defines.h>
-#include "code_80057C60.h"
-#include "math_util_2.h"
 #include "math_util.h"
+#include "math_util_2.h"
+//#include "code_800431B0.h" <- FIXME: create this header file
```

Comment:
maybe use todo as a keyword instead as I think people would tend to search that moreso than 'fixme'